src/anim/controller/anim-controller.js

```diff
@@ -231,34 +231,32 @@ class AnimController {
     }
 
     _transitionHasConditionsMet(transition) {
-        let conditionsMet = true;
-        for (let i = 0; i < transition.conditions.length; i++) {
-            const condition = transition.conditions[i];
+        const conditions = transition.conditions;
+        for (let i = 0; i < conditions.length; i++) {
+            const condition = conditions[i];
             const parameter = this.findParameter(condition.parameterName);
             switch (condition.predicate) {
                 case ANIM_GREATER_THAN:
-                    conditionsMet = conditionsMet && parameter.value > condition.value;
+                    if (parameter.value <= condition.value) return false;
```

Comment:
might be worth putting `condition.value` before `parameter.value` so operators match case statements?

Reply:
I don't think so because the `if` condition must be the negation of the predicate in order to return false (saying the conditions have not been met).

Reply:
How about (for greater than) ?
```javascript
if (!(parameter.value > condition.value)) return false;
```

Seems like having to mentally negate the conditions to make sense of the code is a little unforgiving!

Reply:
Updated for all except the equals based conditions. What do you think?

Reply:
I mean, I *could* use logical not on the equals ones too. Anyway, let me know what you prefer.

Reply:
I've gone ahead and also used logical not outside the equals based conditions. Up to you. 😄

Reply:
Looks good to me!